### PR TITLE
chore: move all packages to the same aws-sdk version ^2.1233.0

### PR DESCRIPTION
**Pull request by evcodes**
#### Description of changes

In the previous releases our binaries were around 175mb and with the latest release candidates, the packages were greater than 250mb leading to a size increase of over 75 mb. This is because there were two different versions of aws-sdk, 2.1113.0 and 2.1169.0. The change was to lock aws-sdk to ^2.1169.0 for all packages.

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [n/a] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [n/a] Relevant documentation is changed or added (and PR referenced)
- [n/a] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [x] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by edwardfoyle — APPROVED** [review]

**Review by sobolk — DISMISSED** [review]

File: packages/amplify-category-storage/package.json
Comment (jhockett) [inline packages/amplify-category-storage/package.json:48]: This is needed, types are pulled in from `packages/amplify-category-storage/src/provider-utils/awscloudformation/import/import-s3.ts` and `packages/amplify-category-storage/src/provider-utils/awscloudformation/import/types.ts`

File: packages/amplify-nodejs-function-template-provider/resources/lambda/crud/package.json.ejs
Comment (jhockett) [inline packages/amplify-nodejs-function-template-provider/resources/lambda/crud/package.json.ejs:14]: This is an ejs template for functions added to an Amplify project. I think changing this is out of scope for this PR.

**Review by jhockett — CHANGES_REQUESTED** [review]
Two important changes are needed, but overall it LGTM.

File: packages/amplify-nodejs-function-template-provider/resources/lambda/crud/package.json.ejs
Comment (jhockett) [inline packages/amplify-nodejs-function-template-provider/resources/lambda/crud/package.json.ejs:2]: The changes to this file should be reverted because we might have snapshot tests that depend on the output, and these spacing changes get reflected in what we generate IIRC.

**Review by jhockett — DISMISSED** [review]
Thanks for addressing my comments, LGTM!

**Review by danielleadams — APPROVED** [review]
LGTM, pending integration (e2e) test run.

**Comment by evcodes** [comment]
Closing because aws-sdk was moved up to 2.1233.0

**Comment by evcodes** [comment]
https://github.com/aws-amplify/amplify-cli/pull/11165

**Pull request closed without merging**
